src/content-helper/dashboard-page/pages/traffic-boost/sidebar/components/links-list/single-link.tsx

```
@@ -73,9 +73,10 @@ export const SingleLink = ( {
 				/>
 			</div>
 			<div className="single-link-details">
-				<div className="single-link-title">
-					{ suggestedPost.title.rendered }
-				</div>
+				<div
+					className="single-link-title"
+					dangerouslySetInnerHTML={ { __html: suggestedPost.title.rendered } }
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)